src/volume_calc.cpp

```diff
@@ -534,8 +534,20 @@ int openmc_calculate_volumes()
 
       // Display domain volumes
       for (int j = 0; j < vol_calc.domain_ids_.size(); j++) {
-        write_message(4, "{}{}: {} +/- {} cm^3", domain_type,
-          vol_calc.domain_ids_[j], results[j].volume[0], results[j].volume[1]);
+        std::string cell_name = "";
```

Comment:
Can I recommend something like this?

```cpp
std::string cell_name {""};

if (vol_calc.domain_type_ == TallyDomain::CELL) {
  int cell_idx = model::cell_map[vol_calc.domain_ids[j]];
  cell_name = model::cells[cell_pos]->name();
  if (cell_name.size()) cell_name.insert(0, " "); // prepend space for formatting
}

 write_message(4, "{}{}{}: {} +/- {} cm^3", domain_type,
   vol_calc.domain_ids_[j], cell_name, results[j].volume[0],
   results[j].volume[1]);
```

Reply:
yes I like it.

Reply:
I was hesitating between the:
`if (vol_calc.domain_type_ == TallyDomain::CELL)`
and 
`if (domain_type == "  Cell ")`

the first one is cleaner, but I felt like the second one might be fast, (not sur speed does matter in that case...)
do you have an opinion about it ?

Reply:
@pshriwise, I implemented you suggestion, it is definitely more concise!
Thx for the review.

Reply:
When possible, I prefer to compare to an enumerator over a string. It also helps in the case that string literal you were comparing to changes in the future.